app-accessibility/flite/flite-2.2-r2.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
```

Comment:
Let's do a new revision please in ~arch. The UB here was horrible and fixing that may expose something.